setup.py

```diff
@@ -43,4 +48,13 @@
         'Topic :: Documentation',
         'Topic :: Software Development :: Documentation',
     ],
+    install_requires= [
+        'Sphinx>=1.4.1',
+    ],
+    cmdclass = {
+        'compile_catalog': babel.compile_catalog,
```

Comment:
@mitya57 An good alternative will be create a requeriments-devs.txt file with the Jinja2 and Babel dependencies or create a TRANSLATE file with this instructions.